design/one-pager-stacks-security-isolation.md

```diff
@@ -56,6 +56,33 @@ From the [Kubernetes documentation](https://kubernetes.io/docs/tasks/access-kube
 Therefore, any time a CRD is created, it will also be at the cluster scope, even if instances of the CRD are scoped to a namespace.
 This has caused confusion in the past, so it's important to clear that up here.
 
+#### Discoverability of Scope
+
+Because CRDs are cluster scoped, but our permission model for stacks is both namespaced and global we will
+ need to label CRDs to indicate which are available in a given context. As the stack-installer processes a
+ cluster or namespace stack install it needs to add labels indicating that it's crossplane environment scoped or
+ namespace scoped to a given namespace.
+
+Labels on a CRD:
+  * crossplane.{scope=namespace|environment}=true
```

Comment:
Nitpick on the labelling, `crossplane.io/scope=namespace` and `crossplane.io/scope=environment` would be more consistent with our current label and annotation conventions, which include `crossplane.io/external-name=foo` and `resourceclass.crossplane.io/is-default-class="true"`.

Reply:
Maybe we should have a quick sync about final wording on all of this stuff. I'm not particularly concerned about choosing one convention over the other, and would agree consistency is probably the best.